point_addition.py

```diff
@@ -2,11 +2,14 @@
 import logging
 from geometric_objects import Triangulation, Triangle, Point
 from incremental_delauney_algorithm import incremental_delauney_algorithm
+from flipping_delauney_algorithm import Flipping_delauney_algorithm
+from convex_hull_triangulation import Convex_hull_triangulation
 
 
 class AlgorithmEnum(enum.Enum):
     FLIPPING = 'Flipping'
     INCREMENTAL = 'Incremental'
+    COVEX_HULL = 'Convex_hull'
```

Comment:
Prosim prejmenuj to na `CONVEX_HULL`, toto lehce drazdi me OCD 😆 + value bych dal jako `Convex hull`. Bude to podle meho vypadat lip v appce.